sia/components/renter.go

```diff
@@ -0,0 +1,14 @@
+package components
+
+import (
+// "github.com/NebulousLabs/Sia/consensus"
+)
+
+type RenterUpdate struct {
+	HostDB HostDB
+}
+
+type Renter interface {
+	// UpdateRenter changes the settings used by the host.
+	UpdateRenter(RenterUpdate) error
+}
```

Comment:
Renter is very far from complete. I've also been waffling on the monolithic style of calling a single update function. It'll definitely have a Rent() function. Tolerance and whatnot will probably be decided on a per-file basis.

Actually, I was thinking file could have high level tags (like 'Chicago', 'High Redundancy', 'Low Latency'; or 'Global', 'High Throughput') and then these would translate into more specific details that the renter can modify itself or can be adjusted through inputs. But that's also all down the line for now we'll just have one renter-controlled setting.